### ISSUE TITLE
Getting wrong type of multivector

### ISSUE THREAD

**Issue by ingodahn** [pygae/galgebra#484]
I am trying to calculate the image of a 3D vector $u=e_1+2e_2+3e_3$ under the rotation with axis vector $e_1+e_2+e_3$. I would expect to get a vector as image but the calculation yields a vector + a tiny multiple of $e_1\wedge e_2\wedge e_3$, perhaps due to rounding errors.

The code is attached
[gaerror.txt](https://github.com/pygae/galgebra/files/10941103/gaerror.txt)

**Comment by eric-wieser** [comment]
Your mistake is that you use `math.pi` which is only an approximation of `sympy.pi`

**Comment by ingodahn** [comment]
No.
1. Using pi=sympy.pi instead of pi=math.pi yields an excess term `2.77555756156289e-17*sqrt(3)*e_1^e_2^e_3` as well
2. Not using pi at all but using the exp() method from the galgebra module yields the same result (see my edit)
Even if your argument would be true, it would only reinforce the problem that the galgebra module is incompatible with the rest of the world, mainly by overloading fundamental operators like *.

**My Use Case**:  In a jupyter notebook with python-sagemath kernel I would like to use galgebra to calculate some geometric objects and visualize the result using SageMath. But when trying the visualization after using galgebra I get the error that the  `vector()` method is unknown. My current workaround is copying the coefficients calculated by galgebra, ignoring excess terms, manually into anothe galgebra-free notebook.

**Comment by eric-wieser** [comment]
Can you share the exact code you're using for 2
?

**Comment by ingodahn** [comment]
Attached is a version that does _not_ produce excess terms by avoiding any use of math. From my other experiments it seems that excess terms are produced as soon as math is involved, for example by using in this script
`
import math

pim=math.cos(math.pi)

u=e_1+2*e_2+(3*pim)*e_3
`

[galgebraTest2.txt](https://github.com/pygae/galgebra/files/11131725/galgebraTest2.txt)

**Comment by eric-wieser** [comment]
Yes, as soon as you use the `math` module you are working with floating-point approximations and not symbolic calculations.

Note you can paste code directly into comments here by using ```` ``` ```` around it

**Comment by Greg1950** [comment]
In GAlgebra's *mv.py* module there are various built-in functions.  Three
of them


*proj(B:Mv, A:Mv)*

    # projects the multivector *A* onto the subspace defined by the blade
*B*
*rot(itheta:Mv, A:Mv)*
    # rotates the multivector *A* through the directed angle defined by the
2-blade *itheta*
*refl(B:Mv, A:Mv)*
    # reflects the multivector *A* through the subspace spanned by the
blade *B*


were added over a decade ago while Alan Macdonald was writing his
textbook *Linear
and Geometric Algebra*.  (Many of that textbook 's problems suggest the use
of GAlgebra.)

Some coding effort would be saved were you to simply use *rot(itheta, A)*.
The answers by other respondents about round-off errors from use of the
module math are still relevant.

**Comment by eric-wieser** [comment]
Indeed, the function referred to by @Greg1950 can be found [in the docs here](https://galgebra.readthedocs.io/en/latest/generated/galgebra.mv.html#galgebra.mv.rot)

**Comment by eric-wieser** [comment]
The meaning of the `hint` argument is described [in the docs for `exp`](https://galgebra.readthedocs.io/en/latest/generated/galgebra.mv.html#galgebra.mv.Mv.exp):

https://github.com/pygae/galgebra/blob/3a53b29fb141be1ae47d8df8fc7005c10869cded/galgebra/mv.py#L1093-L1100
